spec/stats/kernel-selftests/net-14

```diff
@@ -0,0 +1,89 @@
+# selftests: net: udpgro_fwd.sh
+# IPv4
+# No GRO
```

Comment:
thanks, can you confirm from the source code, that it will output this line when iptables is installed correctly?